app/components/candidate_interface/editable_section_warning.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="govuk-inset-text">
   <% if @section_policy.personal_statement? %>
-    Any changes you make to your personal statement will not be included in applications you have already submitted.
+    <%= t('editable_section_warning.personal_statement') %>
```

Comment:
Would it be better if the I18n keys were related to the kind of message rather than the section?

`editable_section_warning.not_included_in_submitted`
`editable_section_warning.included_in_submitted`